### DISCUSSION TITLE
How to send module option to `addServerHandler`'s handler

### DISCUSSION THREAD

**Discussion by adenvt** [nuxt/nuxt#17507] [Questions] [answered]
I have module like this

```ts
export default defineNuxtModule({
  meta: {
    // ...
  },
  setup (options, nuxt) {
    const { resolve } = createResolver(import.meta.url)

    addServerHandler({ route: '/oauth', handler: resolve('./runtime/login.mjs') })
  },
})
```

How access module options inside `./runtime/login` ??

**Comment by pi0** [comment, marked as answer]
Hi @adenvt. For thin, you can push to `nuxt.options.runtimeConfig` as `nuxt.options.runtimeConfig.myModule = { foo: 'bar' }` and then access in the runtime using `useRuntimeConfig()` (`import { useRuntimeConfig } from '#imports'`)

**Reply by adenvt** [reply]
Ok, I'll try it
